src/stores/Blocks.js

```diff
@@ -36,6 +36,18 @@ let Blocks = {
     return state.filter(i => i.parent === parent)
   },
 
+  getDepth(state, block) {
+    let currentBlock = block;
+    let depth = 0
+
+    while (currentBlock || depth == 20) {
```

Comment:
I think it's fine. if we wanted, we could pass a `limit` arg and bail out when that is exceeded. We could pass in `maxDepth`